docs/usage.rst

```diff
@@ -50,6 +50,12 @@ that in some specific cases `XPath`_ is more powerful than `CSS`_.
 .. _XPath: http://www.w3.org/TR/xpath
 .. _CSS: http://www.w3.org/TR/selectors
 
+.. note::
+    HTML comments mentioned within <style> and <script> tags will be treated as raw text.
+    The reason is that it is a common practice to comment out the content of <style> 
```

Comment:
I get it! Now I have reworded it, let me know if this is ideal

`The content of <style> and <script> tags are meant to be application-specific - which is not the concern of an HTML/XML scraper. Thereby, HTML comments mentioned within <style> and <script> tags will be treated as raw text and not as XML elements.`